rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/PluginDescriptor.java

```diff
@@ -64,16 +67,18 @@
 
         private boolean dynamic;
 
+        private int sectionSelector;
+
         Field(String name, String value) {
             this(name, value, "");
         }
 
         Field(String name, String value, String description) {
-            this(name, value, description, false, false, false, false, false);
+            this(name, value, description, false, false, false, false, false, 0);
         }
 
         public Field(String name, String value, String description, boolean password, boolean credential, boolean file,
-                boolean textarea, boolean dynamic) {
+                boolean textarea, boolean dynamic, int sectionSelector) {
```

Comment:
Maybe we should think about  refactoring Field in two classes. Field class growth so much that we now have constructor with 9 arguments to provide. It would be better to group several parameters in a new object that we could build and provide outside the constructor.

Reply:
or use setters (or even builders) rather than constructors ?
The problem of a bit list with similar types is you never know which parameter is what (exception when the IDE tells you)

Reply:
Guys I feel your pain.
But I looked more and I dont see how I can improve it. 
This constructor with 9 arguments is used only at one place.
I mean I can make a builder or extract another class. But all this code will be added for single use.  
Plus we all use IDEA which tells which argument for what. 
This refactoring will bring more code for quite nothing.

Reply:
@Mykhailenko I trust your jugement.